doc/iterations/2025.2/display_diagram_palette_on_right_click.adoc

```diff
@@ -0,0 +1,81 @@
+= Display diagram palette on right click
+
+== Problem
+
+Sirius Web uses the left click to display the palette on diagrams.
+This leads to open the palette on each click, even if the user only wants to select an element.
+
+== Key Result
+
+The palette on diagrams is now enabled through the right click.
+
+=== Acceptance Criteria
+
+- The left click does not display the palette anymore
+- The right click display the palette
+- Escape close the palette as before
+- The left click now closes the palette, as it will create a new selection
+
+=== Scenario
+
+=== Nothing is selected
+
+- Left click on the diagram background: nothing happens.
+- Right click on the diagram background: the diagram palette is opened.
+- Left click on a graphical element: the graphical element is selected, the palette is hidden.
+- Right click on a graphical element: the graphical element is selected, the palette is opened.
+
+=== One graphical element is selected
+
+- Left click on the diagram background: the graphical element is not selected anymore.
+Any existing palette is closed.
+- Right click on the diagram background: the graphical element palette is opened.
+If a creation tool is executed, it will try to create the new graphical element at the position of the palette, possibly far from its graphical container.
+This could lead to extend drastically the size of the graphical container. 
+- Left click on the selected graphical element: nothing happens.
+- Right click on the selected graphical element: the graphical element palette is opened.
+If a creation tool is executed, it will try to create the new graphical element at the position of the palette.
+- Left click on another graphical element (e.g. GE2): GE2 is selected.
+Any existing palette is closed.
+- Right click on another graphical element (e.g. GE2): GE2 is selected.
+The graphical element palette of GE2 is opened.
+If a creation tool is executed, it will try to create the new graphical element at the position of the palette.
```

Comment:
No it will create the element where the click occurs as it is today.

Reply:
Yes